Jenkinsfile.cycle

```diff
@@ -103,15 +104,184 @@ pipeline {
           withEnv(["HOME=${env.WORKSPACE}"]) {
             docker.withRegistry("https://ts-dockerhub.lsst.org/", "nexus3-lsst_jenkins") {
               sh """
-              docker-compose --env-file cycle/cycle.env -f cycle/docker-compose.yaml build ataos atdome atdometrajectory athexapod atmcs_sim atpneumatics_sim atspectrograph hexapod mtdome mtdometrajectory mtm2_sim mtmount ospl-daemon rotator salkafka watcher
+              docker-compose --env-file cycle/cycle.env -f cycle/docker-compose.yaml build \${CONDA_PACKAGES}
               """
               }
           }
         }
         script {
           docker.withRegistry("https://ts-dockerhub.lsst.org/", "nexus3-lsst_jenkins") {
             sh """
-              docker-compose --env-file cycle/cycle.env -f cycle/docker-compose.yaml push ataos atdome atdometrajectory athexapod atmcs_sim atpneumatics_sim atspectrograph hexapod mtdome mtdometrajectory mtm2_sim mtmount ospl-daemon rotator salkafka watcher
+              docker-compose --env-file cycle/cycle.env -f cycle/docker-compose.yaml push \${CONDA_PACKAGES}
+            """
+          }
+        }
+      }
+    }
+    stage ('Build M1M3 simulator') {
```

Comment:
Comment only: A lot of this is repeated code except for a few things that vary. This would be good to put into the shared library once it's available.